content/zh/docs/tasks/observability/logs/otel-provider/index.md

```diff
@@ -0,0 +1,202 @@
+---
+title: OpenTelemetry
+description: 本任务告诉您如何配置 Envoy 代理发送访问日志 OpenTelemetry 收集器。
```

Comment:
```suggestion
description: 此任务向您展示如何配置 Envoy 代理将访问日志打印到其标准输出。
```
Take this title as an example. If you can put some more time, it is easy to learn current trans rather than a MT copy:
https://istio.io/latest/zh/docs/tasks/observability/logs/